exonum-java-binding/fakes/src/main/java/com/exonum/binding/fakes/services/service/TestService.java

```diff
@@ -62,12 +70,33 @@ protected TestSchema createDataSchema(View view) {
   @Override
   public void initialize(Fork fork, Configuration configuration) {
     TestSchema schema = createDataSchema(fork);
-    ProofMapIndexProxy<HashCode, String> testMap = schema.testMap();
+    ProofMapIndexProxy<HashCode, String> testMap = schema.initializeServiceMap();
```

Comment:
If this map is used for a single entry — use Entry (no need for a map and a key).